spec/models/early_career_payments/eligibility_spec.rb

```diff
@@ -78,17 +78,17 @@
       end
 
       [
-        {itt_subject: "Mathematics", itt_academic_year: "2019 - 2020"},
-        {itt_subject: "Mathematics", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Physics", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Chemistry", itt_academic_year: "2020 - 2021"},
-        {itt_subject: "Foreign languages", itt_academic_year: "2020 - 2021"}
+        {itt_subject: "mathematics", itt_academic_year: "2019/2020"},
```

Comment:
Maybe I can use `AcademicYear` throughout this file too?

Reply:
@nomasprime If it is working elsewhere then I think it would be good to be consistent.

If you think it needs some explanation please add appropriate comments. The code base already has good comments where something needs explaining. So I think it would be good to have comments.

Possibly need and update to an ADR?

Reply:
Turns out these need to be serialized DB values too.